jepsen/scalardb/src/scalardb/transfer.clj

```diff
@@ -246,7 +246,7 @@
 (defn consistency-checker
   []
   (reify checker/Checker
-    (check [this test model history opts]
+    (check [this test history opts]
```

Comment:
Is this supposed to be deleted ? If so, can you clarify why ?

Reply:
This `reify` thing is implementing the `Checker` interface, and in the latest Jepsen the interface has been changed to what you see here. I think it is because not all checkers need a `model`, and if one does, it can be constructed with the checker.

Reply:
Got it. Thanks !